doc/deployment_string.md

```diff
@@ -24,6 +24,7 @@ Because Horizon uses the Docker API to start the containers, many of the fields
     - `environment`: `["FOO=bar","FOO2=bar2"]` - environment variables that should be set in the container.
     - `devices`: `["/dev/bus/usb/001/001:/dev/bus/usb/001/001",...]` - device files that should be made available to the container.
     - `binds`: `["/outside/container_path:/inside/container_path1:rw","docker_volume_name:/inside/container_path2:ro"...]` - directories from the host or docker volumes that should be bind mounted in the container. Equivalent to the `docker run --volume` flag. If the first field is not in the directory format, it will be treated as a docker volume. The directory or the docker volume will be created on the host if it does not exist when the containers starts. The last field is the mount options. `ro` means readonly, `rw` means read/write (default).
+    - `tmpfs`: {"/app":""} - There is no source for tmpfs mounts. The following example creates a tmpfs mount at /app
```

Comment:
Please change "The following example" to "It"

Reply:
Hi Ling, I could not figure out how to make the modifications on this pull request, so I modified it in another pull request that will modify this request. I put you as the reviewer for the other pull request.